S/SimpleITK/build_tarballs.jl

```diff
@@ -0,0 +1,78 @@
+using BinaryBuilder, Pkg
+
+name = "SimpleITK"
+version = v"2.2.0"
+
+# Collection of sources required to complete build
+sources = [
+    ArchiveSource("http://github.com/SimpleITK/SimpleITK/releases/download/v$(version)/SimpleITK-$(version).tar.gz", "b07bb98707556ebc2b79aac22dc14950749f509e5b43da8043233275aa55488a")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir
+mount -t tmpfs -o size=12G tmpfs /workspace/srcdir
```

Comment:
```suggestion
```

Reply:
https://github.com/JuliaPackaging/BinaryBuilder.jl/issues/403

this is clearly needed.

Reply:
i was facing a similar issue, so i mounted the workspace/srcdir in the tmpfs to be about 12G, if that make the build work, is that not a good change??

Reply:
> is that not a good change??

No.  You haven't explained _why_ that's needed: https://github.com/JuliaPackaging/Yggdrasil/pull/8721#discussion_r1607191245.

Reply:
https://github.com/JuliaPackaging/BinaryBuilder.jl/issues/403#issuecomment-554551432

Reply:
^

Reply:
![image](https://github.com/JuliaPackaging/Yggdrasil/assets/77094628/0a4303e9-315a-4b86-b09b-1dd794587700)

Reply:
7.8G :(